entities/storobj/storage_object.go

```diff
@@ -44,13 +44,24 @@ func New(docID uint64) *Object {
 	}
 }
 
-func FromObject(object *models.Object, vector []float32) *Object {
+func FromObject(object *models.Object, vector []float32) (*Object, error) {
```

Comment:
I am wondering why FromObject() needs to return an error?
Doesn't FromObject() always return a `nil` error?

Reply:
good spot! I did something differently before which could reuturn an error, but then found a better way. Fixed